backend/tests/test_reservation_check_failure_matrix.py

```
@@ -0,0 +1,132 @@
+        raise SQLAlchemyError("db down")
+
+    def close(self):
+        """No-op close."""
+
+
+@pytest.fixture(scope="module")
+def app():
+    """Build a Flask app with reservation routes for check-endpoint tests."""
+    flask_app = Flask(__name__)
+    flask_app.config.from_object("config.TestConfig")
+    flask_app.config["JWT_SECRET_KEY"] = "test-secret"
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "JWT_SECRET_KEY"

(S105)